setup.py

```diff
@@ -29,6 +32,6 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        "wagtail>=2",
+        "wagtail>=2.15",
```

Comment:
**Suggestion**: see above

```suggestion
        "wagtail>=2.15",
        "django>=3.0,<4.0"
```

Reply:
I think Wagtail 3.0 supports Django4.0, and you can't install wagtail <3.0 with Django4.0 that's why I did this. However, I think we can limit it to Django3 as the priority is to support Wagtail3.0. 

Adding to that, we shouldn't add "Framework :: Django :: 4.0" if we don't support it.